### ISSUE TITLE
KOTLIN IS F*cked, and nobody giving a s#*t about it

### ISSUE THREAD

**Issue by YUGGI200** [Sketchware-Pro/Sketchware-Pro#1381]
**Describing the bug**

look in july month kotlin was totally fine in sketchware pro, kotlin was able to compile properly with any shitty activity that doesn't have any errors, then there comes our PP PranavPurwar with his contribution i reported him this bug on that same day, he replied me that wait till his exams end (before diwali and after diwali end (October end) i waited and after this i tried to contact and get a new date and this man kepts ghosting me, he doesn't reply me in group nor in dm, i just wanna know if he can provide me new date or something, like it's totally becomes his responsibility because he made the contribution that made this bug he's the only man responsible for this bug
man you have 10th school i understand i also have 13th jr college going on but just don't treat kotlin like your abandoned child.

**How to mess kotlin To Reproduce**

my settings of project :
1. AppCompat on
2. Dexer is D8
3. Java 11
4. Disable logcat

1. Make a clean without error kotlin activity in java kotlin manager (Remember Kotlin ACTIVITY without error)

2. Add necessary imports such as any image or resource/layout needed in that activity

3. declare the activity in Android manifest

4. after making sure everything is proper compile the activity and wait till it shows an error on the kotlin compilation page

...

**Expected behavior**
well obviously i expect my activity to be compiled.

**Screenshots**
[Screenshot_2024-11-12-19-33-13-042_pro.sketchware.jpg](https://github.com/user-attachments/assets/66718443-adff-49c7-b305-357981c2b910)

Device and Sketchware Pro-related info:
 - Device: [ Redmi Note 8 pro]
 - OS: [e.g. Android 09/Miui 10.4.4]
 - Sketchware Pro version: [any nightly build that is released after July, but let's take the recent one
v6.4.0-SNAPSHOT-3b543fb-minApi26]

**Additional context**
![kotlin bug.txt](https://github.com/user-attachments/files/17717736/kotlin.bug.txt)

**Comment by laymintu** [comment]
Yes do like this,
 some contributors did the same when i reported bugs, mentioned them 😂🤓🥺

**Comment by YUGGI200** [comment]
> Yes do like this,
>  some contributors did the same when i reported bugs, mentioned them 😂🤓🥺

man that guy made a fucking bug and i talked to him, that dumpster says he is not willing to fix it as not many users reported it, man who tf uses kotlin in sketchware pro

**Comment by CyberXCoder-6649** [comment]
Try downgrading Sketchware-Pro version simple

**Comment by CyberXCoder-6649** [comment]
> > > Yes do like this,
> > > some contributors did the same when i reported bugs, mentioned them 😂🤓🥺
> > 
> > 
> > man that guy made a fucking bug and i talked to him, that dumpster says he is not willing to fix it as not many users reported it, man who tf uses kotlin in sketchware pro
> 
> First of all, be respectful and follow the rules. I cannot just solve a problem when it happens for a single guy and I cannot reproduce the error. Nobody is getting paid here, don't just talk shit. If this oddly random bug which is `you` specific is bothering you, open a PR with a fix. That's how collaborative environment works. Don't go on ranting as if you're paying for it and aren't getting the services.

Yes you are right i was gonna say that too if you are an Android dev then why don't you solve the problem or if you can't solve that problem means it's not always that easy to do and best way is just downgrade to old version simple solution

**Comment by YUGGI200** [comment]
first thing I'm the third person reported 

and second thing 
you
can reproduce it easily, if you follow steps properly - https://t.me/group_LEGENDARY_STREAMER/83523
which you haven't

it's only like you are not willing for it, i came to you because you made it but you are not fixing it so i don't xare about it either, I'll try what i can else i leave it too

**Comment by CyberXCoder-6649** [comment]
> @YUGGI200 You can download and use the previous working version easily, isn't you? These are debug versions and intended for testing purposes only. Therefore you MUST be careful before testing any of these versions.
> 
> You're frustrated for no reason. Don't blame others for your own mistakes.

Nice point, that this is debug version

**Comment by YUGGI200** [comment]
> @YUGGI200 You can download and use the previous working version easily, isn't you? These are debug versions and intended for testing purposes only. Therefore you MUST be careful before testing any of these versions.
> 
> You're frustrated for no reason. Don't blame others for your own mistakes.

let me tell you the real and legitimate reason behind my frustration

it's my fault that i told y'all half story, so how will you feel if you proposed your girlfriend on 27th July 2024 and she said yes, and then on 14th August 2024 you both decided to marry but she says she'll complete her exams first, her exams start on 23 October and ends on month end, but your gf starts ghosting you, and on 11 September she messages you and says let's breakup, how'd you feel man?

pranav did the same to me
on 27 july 2024 this guy made a contribution, which caused this issue and i reported him on the same day, he analyzed it and confirmed it as a bug, on 14 August another guy reported the same bug, he told us to wait until he finishes his exams, and exams in india ends on diwali start, which was different for everyone but as he's in 10th i know his exam ending date it's 20-24 October, then i messaged him on October month end as he's exams got over, this guy ghosted me straight up till 11th September then i had to highlight this issue, and then this dumbo started yapping about it's only happening with me only and not other (obviously it's totally lie), he intentionally says he can't highlight the issue, even if i said the instructions clearly and mentioned that make sure you atleast implement something in that kotlin activity before reproducing, man he was trying to bullshit me

that's why I'm frustrated indeed
